requirements/cython.txt

```diff
@@ -1 +1 @@
-cython==3.0.11
+cython~=3.0.11
```

Comment:
Because this is a constraint file integrated in `cibuildwheel` via `pyproject.toml`, I'd expect it to be pinned exactly.
```suggestion
cython==3.0.11
```

Thought I think I'd just go ahead and merge #1469 in to bump it already and this file wouldn't be updated in the PR.

Reply:
Done! I pinned it to 3.0.11. Once #1469 is in, I can rebase so that we get 3.0.12.